### ISSUE TITLE
Probabilistic Neural Architecture Search

### ISSUE THREAD

**Issue by sile** [sile/readings#94]
URL
-----

https://arxiv.org/abs/1902.05116

Abstract
----------

> In neural architecture search (NAS), the space of neural network architectures is automatically explored to maximize predictive accuracy for a given task. Despite the success of recent approaches, most existing methods cannot be directly applied to large scale problems because of their prohibitive computational complexity or high memory usage. In this work, we propose a Probabilistic approach to neural ARchitecture SEarCh (PARSEC) that drastically reduces memory requirements while maintaining state-of-the-art computational complexity, making it possible to directly search over more complex architectures and larger datasets. Our approach only requires as much memory as is needed to train a single architecture from our search space. This is due to a memory-efficient sampling procedure wherein we learn a probability distribution over high-performing neural network architectures. Importantly, this framework enables us to transfer the distribution of architectures learnt on smaller problems to larger ones, further reducing the computational cost. We showcase the advantages of our approach in applications to CIFAR-10 and ImageNet, where our approach outperforms methods with double its computational cost and matches the performance of methods with costs that are three orders of magnitude larger.

Table of Contents
--------------------

- (1) Introduction
- (2) Related Work
- (3) Methodology
  - (3-1) Architecture search space
  - (3-2) Deterministic approach to architecture search
- (4) Experiments and Results
  - (4-1) Neural architecture search using CIFAR-10
  - (4-2) Architecture fine-tuning using a fully-sized network
  - (4-3) Architecture evaluation on ImageNet
- (5) Conclusion

BibTeX
------

```bibtex
@misc{casale2019probabilistic,
    title={Probabilistic Neural Architecture Search},
    author={Francesco Paolo Casale and Jonathan Gordon and Nicolo Fusi},
    year={2019},
    eprint={1902.05116},
    archivePrefix={arXiv},
    primaryClass={stat.ML}
}
```
